vsutil.py

```diff
@@ -85,3 +86,44 @@ def join(planes: list, family=vs.YUV) -> vs.VideoNode:
     Joins the supplied list of planes into a YUV video node.
     """
     return core.std.ShufflePlanes(clips=planes, planes=[0], colorfamily=family)
+
+
+def getw(h, ar=16 / 9, only_even=True):
+    """
+    returns width for image (taken from kagefunc)
+    """
+    w = h * ar
+    w = int(round(w))
+    if only_even:
+        w = w // 2 * 2
+    return w
+
+
+def is_image(filename: str) -> bool:
+    """
+    Returns true if a filename refers to an image.
+    """
+    return bool(re.search(r'\.(png|jpe?g|bmp|tiff?)$', filename))
```

Comment:
https://docs.python.org/3/library/os.path.html#os.path.splitext
```suggestion
    return os.path.splitext(filename)[-1] in {"png", "jpg", "jpeg", "bmp", "tif", "tiff"}
```

Reply:
Or what about letting the OS tell you if it is an image.

```suggestion
    return mimetypes.types_map[os.path.splitext(filename)[-1]].startswith("image/")
```